### ISSUE TITLE
Change newsletter signup code. Use code below.

### ISSUE THREAD

**Issue by mariobehling** [fossasia/labs.fossasia.org#29]
```html
<form action="http://fossasia.createsend.com/t/t/s/krihdd/" method="post" id="subForm">
    <p>
        <label for="fieldName">Name</label><br />
        <input id="fieldName" name="cm-name" type="text" />
    </p>
    <p>
        <label for="fieldEmail">Email</label><br />
        <input id="fieldEmail" name="cm-krihdd-krihdd" type="email" required />
    </p>
    <p>
        <button type="submit">Subscribe</button>
    </p>
</form>
```

**Comment by ty221** [comment]
Fixed in #30

**Comment by eMBee** [comment]
sorry, i had to revert that because the PR contained a bad/unrelated commit. i  pushed the proper fix instead.
